include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -249,20 +249,87 @@ template<int dim, typename Number>
 void
 SpatialOperatorBase<dim, Number>::distribute_dofs()
 {
+  if(param.spatial_discretization == SpatialDiscretization::L2)
+  {
+    fe_u = std::make_shared<dealii::FESystem<dim>>(dealii::FE_DGQ<dim>(param.degree_u), dim);
+
+    pcout << std::endl
+          << "Discontinuous Galerkin finite element discretization:" << std::endl
+          << std::endl
+          << std::flush;
+  }
+  else if(param.spatial_discretization == SpatialDiscretization::HDIV)
+  {
+    // In-parameter for RT is the degree in tangent direction, i.e the degree in normal direction
+    // will be degree+1. The FE also saves fe_u.degree = degree+1.
```

Comment:
This comment should only explain how the constructor is defined in deal.II, i.e. "The constructor of ... takes the degree in tangential direction as an argument. The degree in normal direction is one degree higher."

Whether ``param.degree_u`` is the degree in normal direction or in tangential direction is in my opinion a convention in ExaDG. We could also define it differently, but then we would need to adjust the number of quadrature points, in order to integrate polynomials of degree ``param.degree_u +1`` in normal direction exactly. However, this information should be given at the relevant places in my opinion, i.e. in the class ``Parameters`` and where we define the quadrature rule.

Reply:
I saw that you added the sentence in the first paragraph of the above comment. Please comment on the second paragraph so that it is a bit easier for me to keep the overview and so that I can mark comments as resolved.